terra-jupyter-base/Dockerfile

```diff
@@ -49,18 +45,16 @@ RUN apt-get update && apt-get install -yq --no-install-recommends \
 
     # install script requirements
     sudo \
-    wget \
     locales \
-    git \
-    jq \
 
     # for ssh-agent and ssh-add
     keychain \
 
-    # for jupyterlab extensions
-    nodejs-dev \
-    node-gyp \
-    npm \
+   # for jupyterlab extensions
```

Comment:
Maybe could just delete this (I bet deep learning containers probably already has node installed since it supports JupyterLab..?)